packages/cms-lib/lib/encoding.js

```diff
@@ -0,0 +1,23 @@
+const base64EncodeString = valueToEncode => {
+  if (typeof valueToEncode !== 'string') {
+    return valueToEncode;
+  }
+
+  const stringBuffer = Buffer.from(valueToEncode);
+  return encodeURIComponent(stringBuffer.toString('base64'));
```

Comment:
I think it is a bit surprising that this is url-encoded in addition to base64 encoded. Isn't the base64 encoded string URL safe?

Reply:
No - it breaks because it sends `=` instead of `%3D` without URL encoding it

Reply:
I guess I was thinking of the `base64url` encoding from https://tools.ietf.org/html/rfc4648. Not a big deal. That said, I do think that `encodeURIComponent` should be called closer to where the URL is constructed. I am also a bit unclear on why base64-encoding is necessary. Why do we need to encode the id of the log?

Reply:
This is a BE requirement in order to retrieve the subsequent log(s).

@bkrainer can you add some color?